scripts/test/index.js

```diff
@@ -0,0 +1,9 @@
+const { spawn } = require('child_process')
+const { inputDir = './source' } = require('yargs').argv
+
+const args = [
+  require.resolve('mocha/bin/_mocha'),
+  `${inputDir}/**/__tests__/*-test.js`
```

Comment:
So this is the same as before... however we're able to lean on expanding paths to things in known locations. Advantage is having it here in a js file rather than an `.opts` file.

```
const defaultOpts = [
  '--recursive',
  '--require',
  join(__dirname, './setup.js'),
  '--compilers',
  `js:${join(__dirname, './compiler.js')}`
]
```